Initial draft of "This Year In Monero - 2021"

sethsimmons opened the pull request:
This is my first post here, but I think it should be pretty much good to go!

Please let me know if you have any further suggestions or any changes that need to be made.

Thanks!

sethsimmons commented:
Netlify preview looks good to me now that image tags are fixed.

Should be ready for review/merge!

erciccione commented on _posts/2021-04-18-this-year-in-monero.md:
You meant "Twitter account"

sethsimmons replied:
Fixed.

erciccione reviewed (changes requested):
Nice post. Two requests:

- Please run the png images through a png compressor like tinypng.com and then resubmit them. It will save a lot of space. For example `monero-hashrate-2021.png` could be shrinked from 330kB to 94kB, a ~72% saving.

- This article is taken from your blog, please add a disclaimer at the bottom of the post with a link to the original article (https://sethsimmons.me/posts/this-year-in-monero-2021/). It will avoid future confusions.

sethsimmons commented:
> Nice post. Two requests:
> 
>     * Please run the png images through a png compressor like tinypng.com and then resubmit them. It will save a lot of space. For example `monero-hashrate-2021.png` could be shrinked from 330kB to 94kB, a ~72% saving.
> 
>     * This article is taken from your blog, please add a disclaimer at the bottom of the post with a link to the original article (https://sethsimmons.me/posts/this-year-in-monero-2021/). It will avoid future confusions.

All changes made and pushed. Will validate that they look alright once the CI runs and provides a preview.

Thanks for the suggestions!

sethsimmons commented:
For future reference https://github.com/kornelski/pngquant was a great tool for this, ended up using that for my own blog post as a test and was able to shrink them even further using:

`pngquant *.png --strip --skip-if-larger --quality=65-80 --verbose --ext=.png --force`

I'll use that in future posts.

erciccione reviewed (commented):
> For future reference https://github.com/kornelski/pngquant was a great tool for this

Good to know. Thanks for the link :)

Just one more thing and then this will be good to go: could you update the file name with today's date? People using RSS feeds will be confused if they receive a feed for a post that appear to have been published a week ago. Nothing important, but since it's a quick fix, better do it.

sethsimmons commented:
> > For future reference https://github.com/kornelski/pngquant was a great tool for this
> 
> 
> 
> Good to know. Thanks for the link :)
> 
> 
> 
> Just one more thing and then this will be good to go: could you update the file name with today's date? People using RSS feeds will be confused if they receive a feed for a post that appear to have been published a week ago. Nothing important, but since it's a quick fix, better do it.

Done! Also added a minor note on the miner count numbers that I had added to my own blog in the meantime.

erciccione reviewed (approved):
Needs to be squashed.
